### ISSUE TITLE
Can't build today - clsql-sys symbol missing?

### ISSUE THREAD

**Issue by quicklisp** [AccelerationNet/clsql-helper#5]
http://report.quicklisp.org/2018-02-09/failure-report/clsql-helper.html#clsql-helper has a log. I get this:

    READ error during COMPILE-FILE: Symbol "TIME-TO-UTC" not found in the CLSQL-SYS package

**Comment by bobbysmith007** [comment]
Ahh... yes, I see I should have delayed releasing this till the clsql version it requires was made.  Apologies, I will roll master back and leave this work in a branch till clsql lands.  Thanks

**Comment by bobbysmith007** [comment]
It should be back in a state where master and clsql are at compatible versions.

**Comment by quicklisp** [comment]
Thanks! And if you're into RSS, try
http://report.quicklisp.org/feeds/author/AccelerationNet.rss for fun
automatic reports!

**Comment by bobbysmith007** [comment]
Yeah, I reset the head to the previous stable state.  Does that mess up the quicklisp workflow?  If so, apologies, I will try to remember to revert rather than force pushing.

What's the best way to rectify this?  Should I put back the patches, then revert them?

**Comment by quicklisp** [comment]
I usually update already-fetched git projects with "git pull", and a force push can make that error out. But I cleared the git cache and rebuilt.

There are still a number of failing minor AccelerationNet systems - the above RSS feed shows more details.

**Comment by xach** [comment]
This happened again - same issue with the same symbol.

**Comment by quicklisp** [comment]
http://report.quicklisp.org/2018-02-26/failure-report/clsql-helper.html#clsql-helper has the current failure log.

**Comment by bobbysmith007** [comment]
Sorry about that, 
I have been trying to switch to using git submodules rather than sibling libraries and I seem to keep messing up what branch I'm on and therefore overwriting master.  

I applied a rollback commit this time instead of resetting, so *hopefully* it works with no more personal intervention.
